reproducibility_project/src/engine_input/mcccs/fort77maker_twobox.py

```diff
@@ -0,0 +1,457 @@
+    # high-precision xyz files written
+
+    lengths1 = filled_boxes[0].box.lengths
+    lengths2 = filled_boxes[1].box.lengths
+    mols = []
+    for mol in filled_boxes[0].children:
+        mols.append(mol.name)
+    num_each_moltype_box1 = list(Counter(mols).values())
+    mols = []
+    for mol in filled_boxes[1].children:
+        mols.append(mol.name)
+    num_each_moltype_box2 = list(Counter(mols).values())
+
+    #######
+    lortho = [True, True]  # nbox elements
+    lconfig_file = [
+        False,
+        False,
+    ]  ## are you providing any config file for your box, config file is written by MCCCS-MN
+    CellVec = []
+    CellLengths = []
+
+    for i in lortho:
+        CellVec.append(np.identity(3))
+
+    for i in range(NBox):
+        if lortho[i]:
+            CellLengths.append(np.ones((1, 3)))
+        else:
+            CellLengths.append(np.identity(3))
+    ###########Change code after this line
+    # you need to input cellvecs (3x3 matrix) if your box is a non ortho box
+    # if your box is non ortho, input your celllengths as 3x3 matrix, else celllengths[i] would be 1x3 np matrix
+    # in this example cellvec[0] is a 3x3 matrix as my zeolite box is non-ortho and I have to define 3x3 cell lengths as well
+    # box 2 is isotropic liq, so it just needs celllengths (1x3)
+    # change your cell vectors and cell sides here
+    CellLengths[0][0][0] = lengths1[0] * 10
+    CellLengths[0][0][1] = lengths1[1] * 10
+    CellLengths[0][0][2] = lengths1[2] * 10
+    CellLengths[1][0][0] = lengths2[0] * 10
+    CellLengths[1][0][1] = lengths2[1] * 10
+    CellLengths[1][0][2] = lengths2[2] * 10
+    ##########################################################
+
+    atom_list = []
+    molecule_names = []
+    nbeads_list = []
+    charge_list = []
+    for i in range(len(molecules)):
+
+        current_molecule = mb.clone(molecules[i])
+        molecule_names.append(current_molecule.name)
+        nbeads_list.append(0)
+        for particle in current_molecule.particles():
+            atom_list.append(particle.name)
+            nbeads_list[-1] += 1
+            charge_list.append(particle.charge)
+
+    AtomsBox = {}
+
+    AtomsBox[1] = atom_list
+    AtomsBox[2] = atom_list
+
+    MoleculesBox = {}
+    MoleculesBox[1] = molecule_names
+    MoleculesBox[2] = molecule_names
+    NBeadsBox = {}
+    NBeadsBox[
+        1
+    ] = nbeads_list  # list that contains number of beads for each molecule type
+    NBeadsBox[2] = nbeads_list
+
+    NMoleculesBox = {}
+    NMoleculesBox[1] = num_each_moltype_box1  # number of molecules
+    NMoleculesBox[2] = num_each_moltype_box2  # number of molecules
+    charge_Box = {}
+    charge_Box[1] = charge_list
+    charge_Box[2] = charge_list
+
+    ####Coordinate file names
+    fileBox = {}
+    fileBox[1] = xyz_file[0]
+    fileBox[2] = xyz_file[1]
+
+    #########config file names
+    config_file = {}
+    # INPUT ENDS HERE
+
+    # INPUT ENDS HERE
+
+    # INPUT ENDS HERE
```

Comment:
Are we expecting the user to edit this script?

Reply:
Thanks! Should be removed. 

I made this script (restart file write for MCCCS) so that other people in the group can also use it for their simulations. And then pasted it here. Will delete these unnecessary lines.